README.md

```diff
@@ -26,7 +26,9 @@ To install ansible-rulebook, we can install our Galaxy Collection, which has a p
 
 Once the Collection is installed, you can run the install_rulebook_cli.yml playbook. The following example would install ansible-rulebook CLI on your local system:
 
-`ansible-playbook -i localhost, -c local ansible.eda.install_rulebook_cli`
+`ansible-playbook -i localhost, -c local ansible.eda.install_rulebook_cli -K`
```

Comment:
`-K` is not required as there are multiple ways to provide the credentials or even it could be not necessary (e.g. a passwordless sudoer)